internal/kafka/internal/presenters/cluster.go

```diff
@@ -18,11 +18,22 @@ func PresentEnterpriseCluster(cluster api.Cluster, fleetShardParams services.Par
 		})
 	}
 
-	c := public.EnterpriseCluster{
+	c := public.EnterpriseClusterRegistrationResponse{
 		ClusterId:            cluster.ClusterID,
 		Status:               cluster.Status.String(),
 		FleetshardParameters: fsoParams,
 	}
 
 	return c, nil
 }
+
+func PresentEnterpriseCluster(cluster api.Cluster) public.EnterpriseCluster {
+	reference := PresentReference(cluster.ID, cluster)
+	return public.EnterpriseCluster{
+		Id:        cluster.ID,
+		Status:    cluster.Status.String(),
+		ClusterId: cluster.ClusterID,
+		Kind:      reference.Kind,
+		Href:      reference.Href,
```

Comment:
this assumes that we will have clusters/{id} endpoint

Reply:
Thanks. It totally make sense. See https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1446#discussion_r1046828146 and https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1446#discussion_r1046825981